README.md

```diff
@@ -1,2 +1,401 @@
-# example-application
-Example out-of-tree application that is also a module
+![Zephyr Project logo](https://github.com/zephyrproject-rtos/zephyr/blob/v1.14-branch/doc/images/Zephyr-Project.png)
+
+
+# Module Overview
+
+The Zephyr Project is a scalable real-time operating system (RTOS) supporting
+multiple hardware architectures, optimized for resource constrained devices,
+and built with security in mind.
+
+The Zephyr build system supports code tree "plug-ins" called *modules*.
+See the [Zephyr Modules (External projects)][5] for a high-level overview
+of the build system module mechanism.
+
+
+## Scope
+
+The Zephyr eXample Application (ZXA) module is supplementary to the Zephyr
```

Comment:
Do we really need a name like "ZXA" for this? Why not just "example application" and "example module" and "example west project" all the way?

Reply:
We need to carve out our portion in each global namespace (e.g. Kconfig, C preprocessor symbols, C functions, include file pathnames, CMake variables)

Reply:
I don't think that is really necessary. I disagree with the global namespace because this might send users the impression that one needs to define one, whereas for the most part this is never required. I would very much prefer that we stick with "example application" and "example module" so that everything is neutral and generic, and let users decide if they want to come up with additional namespaces themselves.

Reply:
For the use case you are trying to support, the namespace is not necessary.  For the one that I am trying to support and that drove this effort, it is.

It seems like your use case could be well served by just enhancing standard Zephyr documentation.  Why does your case need a separate repository?